Assets/BowAndArrowImproved/Scripts/PreWaveBallon.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PreWaveBallon : MonoBehaviour
```

Comment:
why create an entire class for something that does the same thing as a regular balloon?! doesn't make sense at all!

plus, WHY IN GOD'S NAME DOES THIS STUPID SINGLE BALOON ACCESSES GAME MANAGER DIRECTLY?!
we talked about the importance of GameManager and how it's dangerous for random classes to access it directly just yesterday... I don't think we need this class at all, this can be done using a regular balloon, using a script that checks whether a "pop" event occured while we're at the "pre-wave" state

please change it